### PR TITLE
Plutarch OCR corrections; 079-089

### PR DESCRIPTION
**Pull request by helmadik**

### CODE REVIEW

File: data/tlg0007/tlg084a/tlg0007.tlg084a.perseus-grc4.xml [hunk 1]
```diff
@@ -79,235 +79,235 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
...
... ἐκεῖνός ἐστι. τὸν αὐτὸν οὖν τρόπον ὡς ἔοικε <milestone n="20" unit="tlnum" /> καὶ τοῦ παρὰ Ῥωμαίοις ἱερέως ἴδιόν ἐστι τὸ μήθʼ ἵππῳ χρῆσθαι μήτε πλείονας νύκτας ἀποδημεῖν τριῶν μήτʼ ἀποτίθεσθαι τὸν πῖλον, ἀφʼ οὗ καὶ <q>φλᾶμεν</q> κέκληται. πολλὰ δʼ ἄλλα δηλοῦται πᾶσι διὰ τοῦ <milestone n="274d" unit="stephpage" xml:id="stephpage-274d" /> ἱερέως· ὧν ἕν ἐστι καὶ τὸ ἐν ὑπαίθρῳ μὴ ἀλείφεσθαι. <milestone n="25" unit="tlnum" /> τὸ γὰρ ξηραλοιφεῖν ὑφεωρῶντο Ῥωμαῖοι σφόδρα, καὶ τοῖς Ἕλλησιν οἴονται μηδὲν οὕτως αἴτιον δουλείας γεγονέναι καὶ μαλακίας, ὡς τὰ γυμνάσια καὶ τὰς παλαίστρας, πολὺν ἄλυν καὶ σχολὴν ἐντικτούσας <pb xml:id="v.2.p.277" /> ταῖς πόλεσι καὶ κακοσχολίαν καὶ τὸ παιδεραστεῖν καὶ τὸ διαφθείρειν τὰ σώματα τῶν νέων ὕπνοις καὶ περιπάτοις καὶ κινήσεσιν εὐρύθμοις καὶ διαίταις ἀκριβέσιν, ὑφʼ ὧν ἔλαθον ἐκρυέντες τῶν ὅπλων καὶ <milestone n="5" unit="tlnum" /> ἀγαπήσαντες ἀνθʼ ὁπλιτῶν καὶ ἱππέων ἀγαθῶν εὐτράπελοι καὶ παλαιστρῖται καλοὶ λέγεσθαι. ταῦτα γοῦν <milestone n="274e" unit="stephpage" xml:id="stephpage-274e" /> ἔργον ἐστὶν ἀποφυγεῖν εἰς ὕπαιθρον ἀποδυομένους· οἱ δὲ κατʼ οἰκίαν ἀλειφόμενοι καὶ θεραπεύοντες ἑαυτοὺς οὐδὲν ἁμαρτάνουσι. <milestone n="10" unit="tlnum" /> </p> </div>
```
Comment (lcerrato) [inline data/tlg0007/tlg084a/tlg0007.tlg084a.perseus-grc4.xml:163]: Stray characters were introduced here:
`<div n="40" subtype="section" type=I g"textpart"><p rend="indent">`

**Review by lcerrato — CHANGES_REQUESTED** [review]
There is an error in the testing.

**Comment by lcerrato** [comment]
There appears to be an error:
tlg0007.tlg084a.perseus-grc4.xml	 
fatal: Open quote is expected for attribute "type" associated with an  element type  "div". [In (L163 C36)]

**Comment by helmadik** [comment]
oh stupid. Will repair.

**Comment by helmadik** [comment]
@lcerrato Never sure about git but I think the updated file is now in this queue.

**Review by lcerrato — APPROVED** [review]

**Pull request merged**